src/main/java/ee/buerokratt/ruuter/domain/steps/http/HttpDefaultService.java

```diff
@@ -0,0 +1,25 @@
+package ee.buerokratt.ruuter.domain.steps.http;
+
+import ee.buerokratt.ruuter.domain.ConfigurationInstance;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.slf4j.MDC;
+
+import java.util.HashMap;
+
+@Data
+@NoArgsConstructor
+public class HttpDefaultService {
```

Comment:
`DefaultHttpService` sounds a bit better (adjective before noun). I personally would also prefer to use the keyword `Configuration` instead of `service`, seeing as so far we've named all DSL's as `configurations` in code.

Reply:
all "service"/"configuration" will be refactored to "dsl" in future task